examples/nlp/nmt_tutorial.py

```diff
@@ -58,16 +64,42 @@
                                    log_dir=args.work_dir,
                                    create_tb_writer=True,
                                    files_to_copy=[__file__])
-"""Define tokenizer
-In this example, we use YouTokenToMe tokenizer trained on joint
-English & German data for both source and target languages.
 
+# tie weight of embedding and log_softmax layers if use the same tokenizer
+# for the source and the target
+tie_weight = False
```

Comment:
this should be a model argument with argparse

Reply:
@chiphuyen 
tie_weight must be False if the user uses different tokenizers for src and tgt. so we can automatically set it in code.
for en-de, the tokenizer can be the same, if the user passes the same tokenizer, we set tie_weight to True, if not, we set it to False
for en-zh, the src and tgt are very different, we use different tokenizer, so we directly set it to false

Reply:
@chiphuyen 
tie_weight must be False if the user uses different tokenizers for src and tgt. so we can automatically set it in code.
for en-de, the tokenizer can be the same, if the user passes the same tokenizer, we set tie_weight to True, if not, we set it to False
for en-zh, the src and tgt are very different, we use different tokenizer, so we directly set it to false